build-support/src/main/kotlin/app/cash/redwood/buildsupport/KmpTargets.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@file:JvmName("KmpTargets")
+package app.cash.redwood.buildsupport
+
+import org.gradle.api.Project
+import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
+
+@JvmOverloads
+fun addAllTargets(project: Project, skipJs: Boolean = false) {
```

Comment:
Moved this from Groovy to Kotlin so I could add a parameter, `skipJs`. That might have been possible with the include, but I prefer this!

Reply:
Yeah custom build plugins are the full realization of what you're wanting here. I have a branch with it started from a few months ago, but our build is surprisingly complex with lots of little special cases. This is a great start.